src/components/TextPanel/TextPanel.tsx

```diff
@@ -121,7 +128,9 @@ export const TextPanel: React.FC<Props> = ({
               css`
                 flex-grow: 1;
                 overflow: auto;
-              `
+              `,
+              'dt-row-container',
```

Comment:
Should it be `db-container` instead of `db-row-container`?

Reply:
updated